builtin/providers/aws/resource_aws_route_table_test.go

```diff
@@ -497,3 +513,16 @@ resource "aws_route_table" "foo" {
 	propagating_vgws = ["${aws_vpn_gateway.foo.id}"]
 }
 `
+
+const testAccRouteTableConfigPanic = `
```

Comment:
Since this change *fixes* the panic, probably worth having a note either here or on the test above to explain that this was added to cover for an earlier bug, or else the name is somewhat confusing.